apollo-runtime/src/main/java/com/apollographql/android/HttpCacheInterceptor.java

```diff
@@ -0,0 +1,27 @@
+package com.apollographql.android;
+
+import java.io.IOException;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+final class HttpCacheInterceptor implements Interceptor {
+  static final String CACHE_CONTROL_HEADER = "APOLLO-CACHE-CONTROL";
```

Comment:
One option: we can return a `CachingApolloClient` with one of:
```
client.newNetworkCall(operation)
client.newCall(operation,FORCE_NETWORK)
```

@martijnwalraven how does iOS bust through cache?